Remotion/Web/Core/Res/HTML/jquery.iFrameShim.js

```diff
@@ -69,4 +69,4 @@
         });
       }
       : function (s) { return this; };
-})(jQuery);
\ No newline at end of file
+})(Remotion.jQuery);
```

Comment:
we need to make `HtmlHeadAppenderExtensions.RegisterJQueryIFrameShimJavaScriptInclude` inline with with  `HtmlHeadAppenderExtensions.RegisterUtilitiesJavaScriptInclude` while leaving the method as obsolete-as-error with a notsupportedexception in the body. please check other usages of ObsoleteAttribute for examples. => breaking change